default.nix

```diff
@@ -25,15 +26,16 @@ let
     };
     patches = [];
   });
-  llvm = nixpkgs.llvmPackages_13;
-  llvmVersionString = "13.0.0";
+
+  # declares a build environment where C and C++ compilers are delivered by the llvm/clang project
+  # in this version build process should rely only on clang, without access to gcc
```

Comment:
Please add some short explanation why this is important, something like "some dependencies default to gcc if it is present at all, ignoring any settings to the contrary, and we want to avoid that"